config/scale/cluster/samples/delete_filesystem.yaml

```diff
@@ -0,0 +1,7 @@
+# delete the ECE filesystem pieces
+---
+$patch: delete
+apiVersion: scale.spectrum.ibm.com/v1beta1
+kind: Filesystem
+metadata:
+  name: vdisknsd-sample
```

Comment:
And also delete this file completely, since it doesn't apply.

Reply:
Thanks for addressing this!